Cargo.toml

```
@@ -93,6 +93,8 @@ tokio = { version = "1.0.1", features = ["full", "test-util"] }
 hyper = { version = "1.1", features = ["server"] }
 serde_bytes = "0.11.4"
 serde_repr = "0.1.7"
-uuid = { version = "1", features = ["v4"] }
+serde_json = "1.0.68"
+uuid = { version = "1", features = ["v4", "serde"] }
+function_name = "0.3"
```

```suggestion
uuid = { version = "1", features = ["v4", "serde"] }
```
Can these dependencies be skipped?